recipes/channels/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "channels" %}
+{% set version = "2.4.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 08e756406d7165cb32f6fc3090c0643f41ca9f7e0f7fada0b31194662f20f414
+
+build:
```

Comment:
I believe this can be `noarch: python`. See:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python